packages/demo/src/FavoriteColorsForm.re

```diff
@@ -66,7 +68,7 @@ let make = () => {
       <span> {"Name:" |> ReasonReact.string} </span>
       <input
         value={state.values.name}
-        onChange={ReForm.Helpers.handleDomFormChange(handleChange(Name))}
+        onChange={BsReform.Helpers.handleChange(handleChange(Name))}
```

Comment:
why this prefix if you used open on the top? 🤔

Reply:
indeed ended up forgetting 🤔